dll/cpl/mmsys/lang/fr-FR.rc

```diff
@@ -179,39 +179,39 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Multi-canaux"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "By moving the sliders below you can adjust the volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
-    LTEXT "Faible", 9473, 65, 31, 14, 8
+    LTEXT "En déplaçant les curseurs ci-dessous, vous pouvez modifier le niveau du volume de toutes les enceintes connectés à cet ordinateur.", 9506, 8, 4, 212, 36
```

Comment:
"des enceintes". No need to explicit "toutes"

Reply:
"connectées" please

Reply:
```suggestion
    LTEXT "En déplaçant les curseurs ci-dessous, vous pouvez modifier le niveau du volume des enceintes connectées à cet ordinateur.", 9506, 8, 4, 212, 36
```